Student/Student/Notifications/ActivityStreamViewController.storyboard

```diff
@@ -0,0 +1,100 @@
+                                        <constraint firstItem="dvx-D9-Zdi" firstAttribute="leading" secondItem="gT3-JZ-L8G" secondAttribute="leading" constant="16" id="Iyv-4m-Gqd"/>
+                                        <constraint firstItem="qAO-0p-mHa" firstAttribute="top" secondItem="V4E-YT-iVI" secondAttribute="bottom" constant="4" id="J6F-Is-4pe"/>
+                                        <constraint firstAttribute="trailingMargin" secondItem="qAO-0p-mHa" secondAttribute="trailing" constant="16" id="Mke-OU-ZS6"/>
+                                        <constraint firstItem="V4E-YT-iVI" firstAttribute="top" secondItem="k6K-4f-8T0" secondAttribute="bottom" constant="4" id="U8m-E9-T6w"/>
+                                        <constraint firstItem="k6K-4f-8T0" firstAttribute="centerY" secondItem="dvx-D9-Zdi" secondAttribute="centerY" id="ZH2-S9-BM7"/>
+                                        <constraint firstItem="qAO-0p-mHa" firstAttribute="leading" secondItem="k6K-4f-8T0" secondAttribute="leading" id="lFA-Vh-BU0"/>
+                                        <constraint firstAttribute="trailingMargin" secondItem="k6K-4f-8T0" secondAttribute="trailing" constant="16" id="lOp-bJ-vho"/>
+                                        <constraint firstItem="k6K-4f-8T0" firstAttribute="leading" secondItem="dvx-D9-Zdi" secondAttribute="trailing" constant="16" id="nW4-Yg-QVc"/>
+                                        <constraint firstAttribute="bottomMargin" secondItem="qAO-0p-mHa" secondAttribute="bottom" priority="250" constant="4" id="tAD-hw-bxz"/>
+                                        <constraint firstItem="V4E-YT-iVI" firstAttribute="leading" secondItem="k6K-4f-8T0" secondAttribute="leading" id="xzB-ZT-6ry"/>
+                                    </constraints>
+                                </tableViewCellContentView>
+                                <connections>
+                                    <outlet property="courseCode" destination="k6K-4f-8T0" id="yls-Gm-44v"/>
+                                    <outlet property="icon" destination="dvx-D9-Zdi" id="jOK-GG-Uwq"/>
+                                    <outlet property="subTitleLabel" destination="qAO-0p-mHa" id="Lvb-Qp-ZIz"/>
+                                    <outlet property="titleLabel" destination="V4E-YT-iVI" id="qcc-36-rzc"/>
+                                </connections>
+                            </tableViewCell>
+                        </prototypes>
+                        <sections/>
+                        <connections>
+                            <outlet property="dataSource" destination="ecL-16-114" id="hMK-ed-eda"/>
+                            <outlet property="delegate" destination="ecL-16-114" id="9Vz-Le-YPg"/>
+                        </connections>
+                    </tableView>
+                </tableViewController>
```

Comment:
Can you put an empty view in the footer to prevent extra lines from showing up while loading & when the list doesn't take up the whole screen?